packages/web/src/utils/constants.ts

```diff
@@ -9,3 +9,5 @@ export const TIPPING_TOP_RANK_THRESHOLD = 5
 export const MAX_PROFILE_TOP_SUPPORTERS = 5
 export const MAX_PROFILE_SUPPORTING_TILES = 3
 export const MAX_ARTIST_HOVER_TOP_SUPPORTING = 7
+export const NUM_FEED_TIPPERS_DISPLAYED = 2
+export const FEED_TIP_DISMISSAL_TIME_LIMIT = 30 * 24 * 60 * 60 // 30 days
```

Comment:
Should this be in milliseconds instead of seconds?

Reply:
yup!